packages/supabase/lib/src/supabase_stream_filter_builder.dart

```diff
@@ -0,0 +1,152 @@
+part of './supabase_stream_builder.dart';
+
+class SupabaseStreamFilterBuilder extends SupabaseStreamBuilder {
+  SupabaseStreamFilterBuilder({
+    required super.queryBuilder,
+    required super.realtimeTopic,
+    required super.realtimeClient,
+    required super.schema,
+    required super.table,
+    required super.primaryKey,
+  });
+
+  /// Filters the results where [column] equals [value].
+  ///
+  /// Only one filter can be applied to `.stream()`.
+  ///
+  /// ```dart
+  /// supabase.from('users').stream(primaryKey: ['id']).eq('name', 'Supabase');
+  /// ```
+  SupabaseStreamBuilder eq(String column, Object value) {
+    assert(
+      _streamFilter == null,
+      'Only one filter can be applied to `.stream()`',
+    );
```

Comment:
We should be able to get rid of these assertions, right?

Reply:
Good catch!